### DISCUSSION TITLE
Does this project support Xiaomi devices that have HyperOS pre-installed from the factory like Xiaomi 14 series and Redmi K70 series or only HyperOS devices upgraded from MIUI?

### DISCUSSION THREAD

**Discussion by BuiQuangHuy04** [MlgmXyysd/Xiaomi-HyperOS-BootLoader-Bypass#34] [Q&A] [answered]
Does this project support Xiaomi devices that have HyperOS pre-installed from the factory like Xiaomi 14 series and Redmi K70 series or only HyperOS devices upgraded from MIUI?

**Comment by MlgmXyysd** [comment, marked as answer]
Not supported, devices that shipped with HyperOS (e.g. Xiaomi 14, Redmi K70, etc. series) will fail with code 30001.

> Q: Binding failed with error code 30001.
> A: Your device has been forced to verify the account qualification by Xiaomi. Xiaomi lost its 'geek' spirit a long time ago, and there's nothing we can do about it.

**Reply by rustyx** [reply]
Also the global version will fail?

**Comment by jerryzhao2016** [comment]
出厂带澎湃的机器可以用售后刷回MIUI的方法么？
